eisbuk-admin/src/enums/translations.ts

```diff
@@ -128,6 +138,7 @@ export enum DateFormat {
   MonthYear = "Date.MonthYear",
   Time = "Date.Time",
   Placeholder = "Date.Placeholder",
+  Today = "Today",
```

Comment:
Is this used somewhere?

Reply:
Yeah, it is used in the birthday menu and dialog but it wasn't working and I hadn't noticed because it was falling back to this value "Today".
It should instead be "Date.Today".